app/src/js/components/home.js

```diff
@@ -197,12 +183,8 @@ Home.propTypes = {
 export { Home };
 
 export default withRouter(withQueryParams()(connect((state) => ({
-  cumulusInstance: state.cumulusInstance,
-  datepicker: state.datepicker,
-  dist: state.dist,
   executions: state.executions,
   granules: state.granules,
-  pdrs: state.pdrs,
```

Comment:
it wasn't being used at all in the code and wasn't affecting the functionality- when using the class method you can have things declared and defined with values and not use them but still have a working file that shows no lint or functionality errors- when converting to functional components it's a little different- it would work the same way only if there were proper PropTypes given to the variable but still no contribution to the functionality- just thought it would be best if there wasn't anything unnecessary but it totally could've been left in